modules/topojson-j.gradle

```diff
@@ -17,7 +17,7 @@ platform {
 
 	bundle group: 'com.github.wetransform-os', name: 'topojson-j', version: '1.0', {
```

Comment:
There is still version 1.0 referenced here that has dependencies to Geotools 21.
When creating the package imports for OSGi, based on semantic versioning, it is assumed that an update of the major version is not compatible, that's why the imports end up with a constraint requiring version 21 but excluding version 22 or higher.

Reply:
I have actually tried to add as well version 1.1 but the server answer with " Received status code 409 from server".
![image](https://github.com/halestudio/hale-platform/assets/67077116/24ea02f4-b254-480b-b36f-2dcb3c44db73)
and reading your comment

> version = '1.0.0'
> The bundle version should reflect the library version.

I thought that changing this would be enough.

Reply:
> I thought that changing this would be enough.

This is the version you assign to the bundle.
It does not change the dependency.

> I have actually tried to add as well version 1.1

If there are problems with that I would suggest to remove the use of jitpack altogether and use the version published in artifactory (as previously mentioned [here](https://github.com/wetransform-os/topojson-j/pull/4#issuecomment-1853460222)).
That would require to use the version that is declared for the artifact as dependency version (currently a SNAPSHOT version), or there would need to be a proper release of version 1.1 for the topojson library.